lib/manager/bundler/artifacts.ts

```diff
@@ -85,10 +86,21 @@ export async function updateArtifacts(
     logger.debug('No Gemfile.lock found');
     return null;
   }
+
+  if (config.isLockFileMaintenance) {
+    await fs.remove(lockFileName);
+  }
+
   try {
     await writeLocalFile(packageFileName, newPackageFileContent);
 
-    const cmd = `bundle lock --update ${updatedDeps.join(' ')}`;
+    let cmd;
+
+    if (config.isLockFileMaintenance) {
+      cmd = 'bundle install';
```

Comment:
`bundle lock` doesn't work if you have a `Gemfile` and no `Gemfile.lock`?

Reply:
Seems to have done the same. I changed it to `bundle lock` instead.